addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1440,15 +1440,17 @@ export class OdooEditor extends EventTarget {
             ['ordered', 'OL', true],
             ['checklist', 'CL', true],
         ]) {
-            const button = this.toolbar.querySelector('#' + style);
-            if (button && !block) {
-                button.classList.toggle('active', false);
-            } else if (button) {
-                const isActive = isList
-                    ? block.tagName === 'LI' && getListMode(block.parentElement) === tag
-                    : block.tagName === tag;
-                button.classList.toggle('active', isActive);
-            }
+            const buttons = this.toolbar.querySelectorAll('.oe-toggle-' + style);
+            buttons.forEach((button) => {
+                if (button && !block) {
+                    button.classList.toggle('active', false);
+                } else if (button) {
+                    const isActive = isList
+                        ? block.tagName === 'LI' && getListMode(block.parentElement) === tag
+                        : block.tagName === tag;
+                    button.classList.toggle('active', isActive);
+                }
+            });
```

Comment:
> Thanks @stefanorigano, result is cool 👍

Thanks!

>  Wonder why there are additions in `addons/web_editor/static/src/xml/editor.xml` instead of changes (where were the old buttons defined? 🤔)

There are additions to replace the "¶" icon with  the current text style + the new table options section that is visible in the sidebar only.

>  The colorpickers for the text tools are not opening to the bottom or the top anymore. While it is not that annoying when the colorpicker is on the panel's center, it really is when the colorpicker is at the very bottom. (note that I am speaking based on a fix actually being merged: #73947)

Indeed, my changes were prior to that commit. I'll update it, thanks